config/main.py

```diff
@@ -108,7 +108,7 @@ def shutdown_interfaces(ctx, del_intf_dict):
             click.echo("port_dict is None!")
             return False
 
-        if intf in port_dict.keys():
+        if intf in list(port_dict.keys()):
```

Comment:
>list [](start = 19, length = 4)

The same

Reply:
This change was made by the `2to3` tool (see [here](https://www.python.org/dev/peps/pep-0469/#migrating-directly-to-python-3)):

> Note that in many cases d.keys() can be replaced by just d, but the 2to3 migration tool doesn't attempt that replacement.

Plus, keeping `.keys()` makes it clear what the intention is.